tasks/components/SubtaskView.tsx

```diff
@@ -100,6 +100,11 @@ export const SubtaskView = ({
                   }
                 }}
                 onDoneChange={done => {
+                  // taskTemplateId === "" for the creation of a template
+                  if (taskTemplateId) {
```

Comment:
```suggestion
    if (taskTemplateId || taskTemplateId === '') {
      return
    }
```

Reply:
creates are should indicated by an empty string for id and not undefined

Reply:
Yes, but then we still have the problem that the problem occurs when creating the checkbox.

Reply:
> creates should be indicated by an empty string for id and not undefined

Yes, but we should check of !== undefinded because if we have an empty string it's not undefinded